3rdparty/downward/CMakeLists.txt

```diff
@@ -5,18 +5,21 @@ find_package(catkin REQUIRED)
 
 catkin_package()
 
-add_custom_command(
-  OUTPUT installed
-  COMMAND \$\(MAKE\) -f ${PROJECT_SOURCE_DIR}/Makefile
```

Comment:
I see, it is not git. You don't need to add those lines.
Please check [ros build farm](http://www.ros.org/debbuild/indigo.html) after merge it **carefully**.